ios/Breez Notification Service Extension/NotificationService.swift

```diff
@@ -9,7 +9,7 @@ class NotificationService: UNNotificationServiceExtension {
     private var logger: XCGLogger = {
         let logsDir = FileManager
             .default.containerURL(forSecurityApplicationGroupIdentifier: "group.F7R2LZH3W5.com.cBreez.client")!.appendingPathComponent("logs")
-        let extensionLogFile = logsDir.appendingPathComponent("extension-\(Date().timeIntervalSince1970).log")
+        let extensionLogFile = logsDir.appendingPathComponent("ios-extension-\(Date().timeIntervalSince1970).log")
```

Comment:
TODO: Add ios extension logs under `logs/ios-extension/` directory and rename file name to timestamp